2024-05-02 18:42:38 INFO log.py: Yasb - Yet Another Status Bar
2024-05-02 18:42:48 INFO watcher.py: Created file watcher for path C:\ms1\yasb
2024-05-02 19:41:28 ERROR base.py: Failed to execute callback of type 'update_label' with args: []
Traceback (most recent call last):
  File "C:\ms1\yasb\core\widgets\base.py", line 84, in _run_callback
    self.callbacks[callback_type](*callback_args)
  File "C:\ms1\yasb\core\widgets\yasb\file_compare.py", line 75, in _update_label
    if not filecmp.cmp(source, dest):
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\nahid\scoop\apps\python\current\Lib\filecmp.py", line 64, in cmp
    outcome = _do_cmp(f1, f2)
              ^^^^^^^^^^^^^^^
  File "C:\Users\nahid\scoop\apps\python\current\Lib\filecmp.py", line 77, in _do_cmp
    with open(f1, 'rb') as fp1, open(f2, 'rb') as fp2:
                                ^^^^^^^^^^^^^^
PermissionError: [Errno 13] Permission denied: 'C:\\Users\\nahid\\OneDrive\\backup\\arr\\sonarr\\sonarr.db'
